kernel/mount.c

```diff
@@ -299,6 +310,114 @@ static const char* _find_arg(const char* args[], const char* name)
 }
 #endif /* MYST_ENABLE_EXT2FS */
 
+#ifdef MYST_ENABLE_HOSTFS
+static bool _find_mount_source(const char* source, char path[PATH_MAX])
+{
+    bool found = false;
+    for (size_t i = 0; i < _mount_table_size; i++)
+    {
+        if (strcmp(_mount_table[i].source, source) == 0)
+        {
+            found = true;
+            myst_strlcpy(path, _mount_table[i].path, PATH_MAX);
+            break;
+        }
+    }
+    return found;
+}
+
+static long _new_automount_dir(char mountdir[PATH_MAX])
+{
+    long ret = -1;
+    int mount_num = 1;
+
+    myst_strlcpy(mountdir, AUTOMOUNT_DIR, PATH_MAX);
+    if (!_created_automount_dir)
```

Comment:
Again, I would just check for the existence of the directory as described above.

Reply:
Done.